otter/cloud_client.py

```diff
@@ -264,19 +264,20 @@ def _default_throttler(clock, stype, method):
     """
     Get a throttler function with throttling policies based on configuration.
     """
-    policy = {}
-
-    conf = config_value('cloud_client.throttling.create_server_delay')
-    if conf is not None:
-        policy[(ServiceType.CLOUD_SERVERS, 'post')] = _serialize_and_delay(
-            clock, conf)
-
-    conf = config_value('cloud_client.throttling.delete_server_delay')
-    if conf is not None:
-        policy[(ServiceType.CLOUD_SERVERS, 'delete')] = _serialize_and_delay(
-            clock, conf)
-
-    return policy.get((stype, method))
+    cfg_names = {
+        (ServiceType.CLOUD_SERVERS, 'post'): 'create_server_delay',
+        (ServiceType.CLOUD_SERVERS, 'delete'): 'delete_server_delay',
+        (ServiceType.CLOUD_LOAD_BALANCERS, 'get'): 'get_clb_delay',
+        (ServiceType.CLOUD_LOAD_BALANCERS, 'post'): 'post_clb_delay',
+        (ServiceType.CLOUD_LOAD_BALANCERS, 'put'): 'put_clb_delay',
+        (ServiceType.CLOUD_LOAD_BALANCERS, 'delete'): 'delete_clb_delay',
```

Comment:
Hmm. I was hoping we can configure it on [per-tenant basis](http://docs.rackspace.com/loadbalancers/api/v1.0/clb-devguide/content/Rate_Limits-d1e821.html) since CLB doesn't really have scale issues. This may increase the call times and in effect convergence cycle times if many groups are converging. I am not sure if that is an issue. This is definitely a good start for thorttling :)